packages/kbn-test-subj-selector/README.mdx

```diff
@@ -0,0 +1,10 @@
+---
+id: kibDevDocsOpsTestSubjSelector
+slug: /kibana-dev-docs/ops/test-subj-selector
+title: "@kbn/test-subj-selector"
+description: An utility package to quickly get css selectors from strings
+date: 2022-05-19
+tags: ['kibana', 'dev', 'contributor', 'operations', 'test', 'subj', 'selector']
+---
+
+Converts a string from a test subject syntax into a css selectors composed by `data-test-subj`.
```

Comment:
Are we considering adding the top level API to these docs?  I don't have a preference but thought I would ask

Reply:
Good question. My thinking was to add information that help us understand what a given package does. Probably in the future is worth it to add a standard api documentation for each one that exposes a public api.